src/skin/options.html

```diff
@@ -187,6 +187,14 @@ <h3><span class="i18n_import_user_data"></span></h3>
       <h3><span class="i18n_export_user_data"></span></h3>
       <button id="exportTrackers" class="exportButton"><span class="i18n_download"></span></button>
     </div>
+    <div id="reset">
+      <h3><span class="i18n_reset_user_data"></span></h3>
+      <button id="resetUserData" class="resetButton"><span class="i18n_reset"></span></button>
```

Comment:
Do we need these one-off CSS classes? Classes are for grouping ... The ID should be enough.